bindings/nodejs/src/lib.rs

```diff
@@ -47,6 +47,12 @@ fn main(mut cx: ModuleContext) -> NeonResult<()> {
     cx.export_function("callClientMethod", client::call_client_method)?;
     cx.export_function("createClient", client::create_client)?;
     cx.export_function("destroyClient", client::destroy_client)?;
+
+    cx.export_function(
+        "migrateStrongholdSnapshotV2ToV3",
+        client::migrate_stronghold_snapshot_v2_to_v3,
```

Comment:
I've put it in client because stronghold is in client. Not fully convinced it should be here but it doesn't matter much. Let me know if you have a better idea.

Reply:
Put it in secret manager? :trollface:

Reply:
Sure